webapp/src/components/CompetencyRatings.tsx

```diff
@@ -0,0 +1,36 @@
+import { Rating, styled, Typography } from '@mui/material';
+import CircleIcon from '@mui/icons-material/Circle';
+import React from 'react';
+
+const CompetencyRatings = () => {
+  const StyledRating = styled(Rating)({
+    '& .MuiRating-iconFilled': {
```

Comment:
What is this line doing? I'm just curious!

Reply:
If I understand the MUI docs correctly, it's customizing the way we use the `Rating` component from their library. In this case, it's letting us choose the colour fill to be blue instead of yellow, which I believe is the default.

Reply:
We are creating a styled component based on the Rating component here. It was the only way to change the default color (yellow) for the rating to blue. On the 7th line, we use the class name that Material UI created for the circular element.